cirq-core/cirq/ops/pauli_string.py

```diff
@@ -1082,9 +1168,41 @@ def __init__(
         coefficient: 'cirq.TParamValComplex' = 1,
         pauli_int_dict: Optional[Dict[TKey, int]] = None,
     ):
+        """Initializes a new `MutablePauliString`.
```

Comment:
think this needs to be a class level doc to show up on doc website

Reply:
I think after some recent changes both init and class level docstrings do show up on the website. Maybe @MichaelBroughton can confirm

Reply:
Made it class level